chia/server/node_discovery.py

```diff
@@ -310,12 +310,13 @@ async def _connect_to_peers(self, random) -> None:
                     max_tries = 10
                 elif len(groups) <= 5:
                     max_tries = 25
+                sleep_interval = len(groups) * 0.25
+                sleep_interval = min(sleep_interval, self.peer_connect_interval)
+                # Special case: try to find our first peer much quicker.
```

Comment:
I don't get this comment. The special case refers to `len(groups) == 0` right? without this special case, `sleep_interval` would be 0, which would be even *faster*.. so the comment would make more sense to say something like:

> avoid sleeping 0 and busy loop when we don't have any outgoing connections